docs/how-to-guides/develop-a-tool/add-category-and-tags-for-tool.md

```diff
@@ -15,38 +15,49 @@ This document is dedicated to guiding you through the process of categorizing an
 - Please ensure that your [Prompt flow for VS Code](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow) is updated to version 1.1.0 or later.
 
 ## How to add category and tags for a tool
-Run the command below in your tool project directory to automatically generate your tool YAML, use _-c_ or _--category_ to add category, and use _--tags_ to add tags for your tool:
 
-```
-python <promptflow github repo>\scripts\tool\generate_package_tool_meta.py -m <tool_module> -o <tool_yaml_path> --category <tool_category> --tags <tool_tags>
+You can use [pf tool init](../../reference/pf-command-reference.md#pf-tool-init) to initialize a package tool with category and tags:
+```python
+pf tool init --package <package-name> --tool <tool-name> --set category=<tool_category> tags=<tool_tags>
+
 ```
 
-Here, we use [an existing tool](https://github.com/microsoft/promptflow/tree/main/examples/tools/tool-package-quickstart/my_tool_package/yamls/my_tool_1.yaml) as an example. If you wish to create your own tool, please refer to the [create and use tool package](create-and-use-tool-package.md#create-custom-tool-package) guide. 
+Here, we use an example to show the categories and tags of the tool after initialization. Assume that the user executes this command:
```

Comment:
Here we can use the my_tool_1.py as example, also needs to update my_tool_1.py to keep it consistent with the code here

Reply:
update tool script